docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-pytorch.rst

```diff
@@ -0,0 +1,84 @@
+Quantization-aware Training (QAT) with PyTorch
+===============================================
+
+Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with
+PyTorch:
+
+
+1. Apply Post Training Quantization to the model
+##################################################
+
+Quantize the model using the :doc:`Post-Training Quantization <../quantizing-models-post-training/basic-quantization-flow>` method.
+
+.. tab:: PyTorch
```

Comment:
since these blocks are on the dedicated page, let's remove the .. tab:: PyTorch header completely and move    .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
      :language: python
      :fragment: [quantize]
part to the beginning of the line (3 spaces back)
same goes for TF doc

Reply:
Done